shared/actions/favorite.js

```diff
@@ -79,6 +79,7 @@ const folderToProps = (folders: Array<Folder>, username: string = ''): FolderPro
     showingPrivate: true,
     smallMode: false,
     showComingSoon: !flags.tabFoldersEnabled,
+    username,
```

Comment:
I was getting flow errors without it here due to this function returning FolderProps.